test/from-draft6/to-draft7/schema.json

```diff
@@ -0,0 +1,73 @@
+[
+    {
+        "title": "Simple matching schema",
+        "from": {
+            "$schema": "https://json-schema.org/draft-06/schema"
+        },
+        "to": {
+            "$schema": "https://json-schema.org/draft-07/schema"
```

Comment:
Because in theory the "official" ones have an empty fragment, I think we should transform to that. If the user passes a meta-schema without it, that's fine, we can still handle it, but I think we should always output 100% compliant ones!

Reply:
Okay cool! I got it

Reply:
You want to do something like https://github.com/json-schema-org/upgrade-downgrade-rules/pull/36/files here too